packages/react/README.md

```diff
@@ -2,188 +2,63 @@
-import fortmaticModule from '@web3-onboard/fortmatic'
-import torusModule from '@web3-onboard/torus'
-import keepkeyModule from '@web3-onboard/keepkey'
-import dcentModule from '@web3-onboard/dcent'
+import { ethers } from 'ethers'
 
 const injected = injectedModule()
-const coinbase = coinbaseModule()
-const walletConnect = walletConnectModule()
-
-const portis = portisModule({
-  apiKey: 'b2b7586f-2b1e-4c30-a7fb-c2d1533b153b'
-})
-
-const fortmatic = fortmaticModule({
-  apiKey: 'pk_test_886ADCAB855632AA'
-})
 
-const torus = torusModule()
-const ledger = ledgerModule()
-const keepkey = keepkeyModule()
+const infuraKey = '<INFURA_KEY>'
+const rpcUrl = `https://mainnet.infura.io/v3/${infuraKey}`
 
-const trezorOptions = {
-  email: 'test@test.com',
-  appUrl: 'https://www.blocknative.com'
-}
-
-const trezor = trezorModule(trezorOptions)
-
-const dcent = dcentModule()
-
-const web3Onboard = init({
-  wallets: [
-    injected,
-    coinbase,
-    ledger,
-    trezor,
-    walletConnect,
-    keepkey,
-    fortmatic,
-    portis,
-    torus,
-    dcent
-  ],
+// initialize Onboard
+init({
+  wallets: [injected],
   chains: [
     {
       id: '0x1',
       token: 'ETH',
       label: 'Ethereum Mainnet',
-      rpcUrl: 'https://mainnet.infura.io/v3/ababf9851fd845d0a167825f97eeb12b'
-    },
-    {
-      id: '0x3',
-      token: 'tROP',
-      label: 'Ethereum Ropsten Testnet',
-      rpcUrl: 'https://ropsten.infura.io/v3/ababf9851fd845d0a167825f97eeb12b'
-    },
-    {
-      id: '0x4',
-      token: 'rETH',
-      label: 'Ethereum Rinkeby Testnet',
-      rpcUrl: 'https://rinkeby.infura.io/v3/ababf9851fd845d0a167825f97eeb12b'
-    },
-    {
-      id: '0x89',
-      token: 'MATIC',
-      label: 'Matic Mainnet',
-      rpcUrl: 'https://matic-mainnet.chainstacklabs.com'
+      rpcUrl
     }
-  ],
-  appMetadata: {
-    name: 'Blocknative',
-    icon: '<svg><svg/>',
-    description: 'Demo app for Onboard V2',
-    recommendedInjectedWallets: [
-      { name: 'MetaMask', url: 'https://metamask.io' },
-      { name: 'Coinbase', url: 'https://wallet.coinbase.com/' }
-    ]
-  },
-  apiKey: 'xxx387fb-bxx1-4xxc-a0x3-9d37e426xxxx'
-  notify: {
-    enabled: true,
-    transactionHandler: transaction => {
-      console.log({ transaction })
-    }
-  },
-  accountCenter: {
-    desktop: {
-      position: 'topRight',
-      enabled: true,
-      minimal: true
-    },
-    mobile: {
-      position: 'topRight',
-      enabled: true,
-      minimal: true
-    }
-  },
-  i18n: {
-    en: {
-      connect: {
-        selectingWallet: {
-          header: 'custom text header'
-        }
-      },
-      notify: {
-        transaction: {
-          txStuck: 'custom text for this notification event'
-        }
-      }
-    },
-    es: {
-      transaction: {
-        txRequest: 'Su transacción está esperando que confirme'
-      }
-    }
-  }
+  ]
 })
 
 function App() {
   const [{ wallet, connecting }, connect, disconnect] = useConnectWallet()
```

Comment:
When I was writing docs for the new hooks I wasn't sure the best way to note hooks in the example code. I eventually went with importing all and only showing in action of a couple, with the thinking if they copy/paste a dev has them imported and ready but not a ton of extra code to deal with. I can also understand this approach but it requires more reading of the docs to get the other hooks which can be a challenge for some 🤷‍♂️

Reply:
Yeah I agree, it is a bit of a trade off. We can start by simplifying the quick-start for now. I think in the longer term we probably want to rewrite the docs so that it includes a step by step walkthrough of how Onboard helps you solve common web3 app problems (connecting a wallet, tracking the connected addresses, switching networks), combined with a straight API that is derived from our types. That way we cover it in depth, or a dev can just take a quick look at the types for what they need.